### DISCUSSION TITLE
New 3DS controls

### DISCUSSION THREAD

**Discussion by techboy04** [nzp-team/nzportable#98] [General]
Hey, ive been wanting to use the C stick for camera movement and the circle for player movement but the controls customization wouldnt detect either, I tried to transfer my Quake config which has those from ctrQuake but the camera movement seems to be tied to the circle pad

Will it be possible for yall to have a seperate config for New 3DS to allow those controls to be possible?

**Comment by MotoLegacy** [comment]
implemented with https://github.com/nzp-team/glquake/pull/2
